src/js/helpers/ServiceSchemeUtil.js

```diff
@@ -0,0 +1,37 @@
+const BASE_SCHEME_LABEL = "MARATHON_SERVICE_SCHEME";
+const DEFAULT_SCHEME = "http";
```

Comment:
FAR: make it configurable from runtime.js

Reply:
Good point!